common/changes/miwhea-fabric-core-6_2017-02-07-22-38.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Update to Fabric Core 6.0.0",
+      "type": "patch"
```

Comment:
Actually, turns out that if you only have office-ui-fabric-react in your package.json, you won't get a copy of office-ui-fabric-core when you run `npm install`. Think this is because it's under `devDependencies` instead of `dependencies`. So there shouldn't be anyone who's accessing the mixins/variables in Core via React, right?